Example/SFFeatureFlag/FeatureFlagManager.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  FeatureFlagManager.swift
+//  SFFeatureFlag_Example
+//
+//  Created by Maja Sapunova on 22.6.21.
+//  Copyright © 2021 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import SFFeatureFlag
+
+enum FeatureType: String {
+    case profileDetails = "profile_details"
+    case recommendedArticles = "recommended_articles"
+}
+
+/// Wrapper class around the SFFeatureFlag implementation
+class FeatureFlagManager {
```

Comment:
Shouldn't this be marked as public or the file which starts and fetches remote config?
Being in a pod and not public I think it won't be accessbile.

Reply:
FeatureFlagManager class is a wrapper around the SFFeatureFlag implementation and it's not part of the pod, but part of the main application. No need be marked as public

Reply:
The same applies for the FirebaseRemoteConfigService class. We agreed that Firebase won't be added as a dependency to the pod and the all the implementation related with Firebase will be part of the main application